src/azure-cli/azure/cli/command_modules/azure-cli-netappfiles/HISTORY.rst

```diff
@@ -0,0 +1,9 @@
+.. :changelog:
+
+Release History
+===============
+
+2.0.0
+++++++++++++++++++
+
+* GA release.
```

Comment:
There should not be a HISTORY.rst file in the module any longer since this won't be released as a separate Python package. This entry should go in the azure-cli package's HISTORY.rst.

Additionally, the path should not be `command_modules/azure-cli-netappfiles`. The path should be `command_modules/netappfiles`.

Reply:
Ah ok, I checked the 'master' history and did not see sections for each file so I wasn't sure where history should go. I will add this.